src/app/components/menu/Menu.jsx

```diff
@@ -3,45 +3,62 @@ import Box from '@mui/material/Box';
 import Drawer from '@mui/material/Drawer';
 import Button from '@mui/material/Button';
 import List from '@mui/material/List';
-import Divider from '@mui/material/Divider';
 import ListItem from '@mui/material/ListItem';
 import ListItemButton from '@mui/material/ListItemButton';
 import ListItemIcon from '@mui/material/ListItemIcon';
 import ListItemText from '@mui/material/ListItemText';
-import InboxIcon from '@mui/icons-material/MoveToInbox';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+import MonetizationOnIcon from '@mui/icons-material/MonetizationOn';
+import WidgetsIcon from '@mui/icons-material/Widgets';
 import MailIcon from '@mui/icons-material/Mail';
 import MenuIcon from '@mui/icons-material/Menu';
+import useSWR from 'swr';
 
-export default function TemporaryDrawer() {
+const fetcher = (url) => fetch(url).then((res) => res.json());
+
+const Menu = () => {
   const [open, setOpen] = useState(false);
 
   const toggleDrawer = (newOpen) => () => {
     setOpen(newOpen);
   };
 
+  const {
+    data: tabs,
+    isLoading,
+    isError: error,
+  } = useSWR(`${process.env.NEXT_PUBLIC_STRAPI_API_URL}/tabs`, fetcher, {
+    revalidateOnFocus: false,
+    revalidateOnReconnect: false,
+  });
+
+  if (error) {
+    return <p>Failed to fetch</p>;
+  }
+
+  if (isLoading) {
+    return <p>Loading tabs...</p>;
+  }
+
+  const icons = [
+    <ShoppingCartIcon />,
+    <WidgetsIcon />,
+    <MonetizationOnIcon />,
+    <MailIcon />,
+  ];
+
   const DrawerList = (
     <Box sx={{ width: 250 }} role='presentation' onClick={toggleDrawer(false)}>
       <List>
-        {['Inbox', 'Starred', 'Send email', 'Drafts'].map((text, index) => (
-          <ListItem key={text} disablePadding>
-            <ListItemButton>
-              <ListItemIcon>
-                {index % 2 === 0 ? <InboxIcon /> : <MailIcon />}
-              </ListItemIcon>
-              <ListItemText primary={text} />
-            </ListItemButton>
-          </ListItem>
-        ))}
-      </List>
-      <Divider />
-      <List>
-        {['All mail', 'Trash', 'Spam'].map((text, index) => (
-          <ListItem key={text} disablePadding>
+        {tabs?.data.map((tab, index) => (
+          <ListItem key={tab.attributes.Text} disablePadding>
```

Comment:
te tabsy to do przemyślenia czy chcesz na pewno z bazy - ja uznałbym że nie chcesz :P

Reply:
Tak jak rozmawialiśmy i usunęłam pobieranie z bazy